lib/resources/pip.rb

```diff
@@ -75,7 +81,7 @@ def pip_cmd
           return nil
         end
       end
-      pipcmd || 'pip'
+      pipcmd = @virtualenv_path.nil? ? 'pip' : @virtualenv_path
```

Comment:
[nit] I think to keep the logic that was there before -- this would maybe better be something like this:
```ruby
pipcmd || @virtualenv_path || 'pip'
```
So, the order of precedence would be: if the `pip` is found (windows), take that. Otherwise, take `@virtualenv_path`; fall back to 'pip'.

Makes me wonder if this shouldn't be like this, then:
```ruby
@virtualenv_path || pipcmd || 'pip'
```